### ISSUE TITLE
[BUG] Broken on LightDM, the unlock time text adds newlines instead of updating itself

### ISSUE THREAD

**Issue by qoijjj** [34N0/pam-authramp#50]
## Steps to reproduce:

Use authramp with LightDM / Cinnamon
enter incorrect passwords until lockout

## Expected behavior:

on lockout, a timer should be displayed

## Actual behavior: 

on lockout, the timer text is displayed, but isn't updated. instead, updates are sent as newlines like so:

```
Account locked! Unlocking in 30 seconds.
Account locked! Unlocking in 29 seconds.
Account locked! Unlocking in 28 seconds.
```
And then trails off the bottom of the UX.

Please confirm that you can repro.

**Comment by 34N0** [comment]
PAM modules provide a pointer to a Data Buffer containing strings as the PAM conversation. This might be an upstream issue where instead of updating the text with the newest data in the buffer the whole buffer is displayed.

**Comment by 34N0** [comment]
This was fixed in #55 . Tested on v0.9.7
